osbs-client.spec

```diff
@@ -145,7 +127,7 @@ This package contains osbs Python 2 bindings.
 
 
 %prep
-%setup -qn %{name}-%{commit}
+%setup -q
```

Comment:
Could we use autosetup here? (nitpick)

https://rpm.org/user_doc/autosetup.html

Reply:
I;d do it in separate PRs, I;d like to keep it in sync with other projects